tools/setup.bat

```diff
@@ -0,0 +1,16 @@
+;@Findstr -bv ;@F "%~f0" | powershell -Command - & pause & goto:eof
```

Comment:
rename to setupHemtt.bat?

Reply:
Pressed the merge button just as you posted this.

I don't see a reason to rename though, it's documented now and HEMTT should become primary build tool anyways.